expansion/src/main/java/no/unit/nva/expansion/model/ExpandedDoiRequest.java

```diff
@@ -17,7 +16,7 @@
 import nva.commons.apigateway.exceptions.NotFoundException;
 import nva.commons.core.JacocoGenerated;
 
-@JsonTypeName(TYPE)
+@JsonTypeName(ExpandedDoiRequest.TYPE)
```

Comment:
Unintentional?

Reply:
no. it is just to be sure we are not importing some other TYPE constant.